pslab/external/BMP180.py

```diff
@@ -1,32 +1,24 @@
-from __future__ import print_function
-
 import time
-
 from numpy import int16
```

Comment:
This is quite likely unnecessary, I guess? Some Pythonista help with that would be great. :)